plugins/web/opentelemetry-instrumentation-document-load/test/documentLoad.test.ts

```diff
@@ -35,12 +35,15 @@ import {
   StackContextManager,
 } from '@opentelemetry/sdk-trace-web';
 
-import * as assert from 'assert';
+// @ts-expect-error: not an export, but we want the prebundled version
+import chai from 'chai/chai.js';
```

Comment:
ideally we'd just get `assert` to work in the browser, but for some reason it gives an error

Reply:
We used the webpack node polyfills for assert (see https://github.com/open-telemetry/opentelemetry-js-contrib/blob/434cab2539f35e00c4788cb48953b622950122dc/webpack.node-polyfills.js#L3-L6).

IMO not using node polyfills for tests that are only intended for the browser is a good idea - I'd be fine for this to switch to `chai`, would like to get an opinion on it by @martinkuba or @pkanal first though as they're the subject matter experts 🙂